### ISSUE TITLE
How to contact @gorhill?

### ISSUE THREAD

**Issue by Balls0fSteel** [gorhill/uBlock#146]
Since the profile says "file an issue" - here it is.

**Comment by CoolSilver** [comment]
No.

**Comment by gorhill** [comment]
It means: just ask whatever you want to ask here. The project is public, I am not answering any private email about it.

**Comment by alejandrolemus** [comment]
@GotDemBallsOfSteel 
I can confirm he doesn't, already tried ;)

**Comment by GotDemBallsOfSteel** [comment]
@alejandrolemus Oh well. Let the drama continue! :+1: (Time to restock from the shop, I guess.)

**Comment by gorhill** [comment]
What does "drama" have to do with this? The project is developed in the open, so whatever you want to ask me about the project should be in the open. If you feel it needs to ask something privately, it's not to the project's benefit. It's that simple.

**Comment by alejandrolemus** [comment]
@GotDemBallsOfSteel @gorhill 
Oh sorry, I was merely confirming that everyone who has been following this project for some time, knows that if you need to contact the developer about the project, public issues is the way. No interest on drama here. Too much time goes wasted in that kind of stuff here already.

To clarify, I sent an email to @gorhill for some privacy-related issue. Didn't receive a reply, nor was I waiting for one. Maybe is time to follow the new rule for posting: no more +1, me too or I can confirm messages. Apologies everybody.

**Comment by gorhill** [comment]
For the record, I used to respond to some private emails re. github projects a long time ago but I had to stop this, or else it can get out of control. I decided there would be now a very strict separation between my github projects and me as a private person.

So I decided a while ago I won't answer one single private email about my github projects. All there is to know is in here, and if not then an issue should be opened, and of course I will address the issue if and only if it's directly related to the projects.

@alejandrolemus Yes I did receive your email, I appreciate the concern. That's the kind of email that I do not frown upon, as it's about _warning_ me about me leaking private information and not _asking_ me something private. The answer to your email is: that email is public, so if someone really wants to find it, they easily can, so I didn't see it as something to fix.
